### DISCUSSION TITLE
About TextField in ComposeView floating window

### DISCUSSION THREAD

**Discussion by Xiue233** [dofire/Floating-Bubble-View#62] [Q&A] [answered]
Before using this wonderful repository, I can not call my IME out when I wanna input into my TextField in floating window. I have tried to copy the ComposeLifecycleOwner to my project and done anything I guess. Howerver,  it doesn't work anyway in my project.

So please tell me what is the key to enable TextField to be inputable. Thank you so much.

**Comment by dofire** [comment]
@Xiue233 Not sure why the IME doesn't shown in your case, it's not a problem in the demo app. Make sure that you don't include `WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE` flag in your `LayoutParams`

![xs-Screenshot_2024-03-12-08-28-11-706_com mi android globallauncher (Small) (Custom)](https://github.com/TorryDo/Floating-Bubble-View/assets/85553681/45f55cbd-ea24-4305-9522-638040782b5c)

**Reply by Xiue233** [reply, marked as answer]
Centainly it is not caused by the flag. The IME can show when it is called by native view EditText. 
Thank you all the same. Maybe the reason is that I use a third-party repo with terrible Compose support.
